neo4j/hatch.toml

```diff
@@ -12,6 +12,7 @@ matrix.version.env-vars = [
   { key = "NEO4J_VERSION", value = "4.2.14", if = ["4.2"] },
   { key = "NEO4J_VERSION", value = "4.3.10", if = ["4.3"] },
   { key = "NEO4J_VERSION", value = "4.4.3", if = ["4.4"] },
+  { key = "NEO4J_VERSION", value = "5.5.0", if = ["5.5"] },
```

Comment:
Let's change these to `NEO4J_IMAGE` in order to only require one `dd_environment` fixture